packages/@uppy/companion/src/standalone/helper.js

```diff
@@ -32,19 +32,19 @@ const getConfigFromEnv = () => {
     providerOptions: {
       google: {
         key: process.env.COMPANION_GOOGLE_KEY,
-        secret: process.env.COMPANION_GOOGLE_SECRET
+        secret: typeof process.env.COMPANION_GOOGLE_SECRET_FILE === 'string' ? fs.readFileSync(process.env.COMPANION_GOOGLE_SECRET_FILE).toString() : process.env.COMPANION_GOOGLE_SECRET
```

Comment:
can we pull this into a function instead? And reuse it for others too?

Reply:
might be a personal snag but i prefer `('COMPANION_GOOGLE_SECRET_FILE' in process.env)` as a check for this (whether or not we move it into a function and reuse)

Reply:
We sure can!